doc/Type/Seq.pod6

```diff
@@ -57,9 +57,10 @@ B<Caution:> No program should ever assume a C<Seq> may only be iterated once
 even if not cached by the program. Caching is a volatile state exposed to the
 developer as an optimization. The C<Seq> may become cached by many operations,
 including calling C<perl> on the C<Seq> (if called prior to a non-cached
-iteration). From version 6.d, you C<.perl> can be called on consumed <Seq>. If a
-program assumes a Seq can only iterate once, but then is later changed to call
-one of these operations during the loop, that assumption will fail.
+iteration). From version 6.d, you C<.raku> (C<.perl> before version 2019.22) can
```

Comment:
there's something missing between you and `.raku`

Reply:
Also version 2019.11, not 22

Reply:
I think the word 'you' can probably be removed, what do you think? Also, I got this piece of information on version number from https://github.com/Raku/doc/pull/3387#discussion_r422661364 . Can you change the version in that comment to the right version?

Reply:
That was my bad. Done.